app/views/upload.scala.html

```diff
@@ -1,6 +1,47 @@
 @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span>
+            <a href="/dashboard" class="govuk-back-link">Back</a>
+            <span class="govuk-caption-xl">Step 3 of 3</span>
+            <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
+        </span>
+        <div class="govuk-form-group">
+            <label class="govuk-label" for="file-upload-1">
+              Select a folder
```

Comment:
One more thing, sorry: we're adding user-facing text to the [messages file](https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/conf/messages) so that it's easier to review and update it all in one place. @techncl should be able to help with this one because he's working on it at the moment. 🙂